### DISCUSSION TITLE
Where and how to declare global variables?

### DISCUSSION THREAD

**Discussion by FatShibe97** [enable3d/enable3d#192] [Help / Questions] [answered]
Hi, I'm currently using the enable3d-phaser-project-template. I would like to declare a global variable named "score" for use in functions other than create() in mainScene.ts. How and where can I do that? Thanks!

**Comment by yandeu** [comment, marked as answer]
You can do it however you prefer. You are not tied to any specific way of doing it.

_The template hasn't been updated in a while. Make sure you do an `npm update` and report any potential issue you experience._
